src/NodeCompiler/Exception/UnableToCompileNode.php

```diff
@@ -15,6 +15,12 @@
 
 class UnableToCompileNode extends LogicException
 {
+    /**
+     * @psalm-readonly
+     * @var string|null
+     */
+    public $constantName;
```

Comment:
Seems to fail with:

```
ERROR: InaccessibleProperty - src/NodeCompiler/Exception/UnableToCompileNode.php:63:9 - Roave\BetterReflection\NodeCompiler\Exception\UnableToCompileNode::$constantName is marked readonly (see https://psalm.dev/054)

        $exception->constantName = $constantName;
```

I think it should be set in the ctor...

Reply:
Perhaps an accessor is better, then?

`UnableToCompileNode` is not marked as `@internal`, and overriding the constructor might be a backward-compatibility issue (unless I add `$constantName` as an additional parameter.

What do you think?